### PR TITLE
Document object_mk

### PR DESCRIPTION
**Pull request by zrnorth**
My first pr to a decomp project :)

### CODE REVIEW

**Review by AngheloAlf — COMMENTED** [review]
Good job on your first contribution :partying_face: 

Also, take in consideration that we are trying to get everybody who has contributed / wants to contribute to MM decomp to agree to licensing the repo before merging PRs from people who has not yet agreed to it. It is better explained here: https://github.com/louist103/mm-licensing/issues/1.
Could you please read about it and post in that issue if you agree or not. Thanks!

**Comment by zrnorth** [comment]
Just left a comment there.

**Review by AngheloAlf — APPROVED** [review]

**Review by jpburnett — APPROVED** [review]

File: assets/xml/objects/object_mk.xml
Comment (engineer124) [inline assets/xml/objects/object_mk.xml:7]:
```suggestion
        <Animation Name="gMarineResearcherYellAnim" Offset="0x7D8" />
        <Animation Name="gMarineResearcherShakeInFearAnim" Offset="0x10F4" />
        <Animation Name="gMarineResearcherStrokeChinAnim" Offset="0x1964" />
```
We tend to avoid `ing` in anim names like this

Comment (zrnorth) [reply assets/xml/objects/object_mk.xml]: Gotcha. Changed to your suggestion

**Comment by hensldm** [comment]
Looks like you need to run the formatter. `./format.py -jN` where `N` is the number of jobs you want to run. Note it requires `clang-format-14`.

**Comment by zrnorth** [comment]
> Looks like you need to run the formatter. `./format.py -jN` where `N` is the number of jobs you want to run. Note it requires `clang-format-14`.

Thanks for the heads up! I ran it and made a new commit, hopefully the checks pass now.

**Comment by zrnorth** [comment]
Anything else I need to do for approval? Lmk if I missed anything 😄

**Review by engineer124 — APPROVED** [review]
Great work on your first PR!

**Pull request merged**
